bundles/org.openhab.binding.solarmax/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -37,10 +37,15 @@
 				<description>Hostname or IP Address</description>
 			</parameter>
 			<parameter name="portNumber" type="integer" required="false">
-				<label>Port</label>
+				<label>Port Number</label>
 				<description>Port Number (defaults to 12345)</description>
 				<default>12345</default>
 			</parameter>
+			<parameter name="deviceAddress" type="integer" required="false">
+				<label>Device Address</label>
+				<description>Device Address (defaults to 1)</description>
```

Comment:
In the README description is: "Device address for devices connected serially." Consider if you want to bring some of that extended description here, as right now it just duplicates the label and mentions default value.

Reply:
Done